.github/workflows/acceptance.yml

```diff
@@ -8,11 +8,23 @@ on:
     tags: [v*]
 
 jobs:
-  api:
-    name: API
+  api_chunk_1:
```

Comment:
Do the chunks have any logic to the grouping?
If so let's name it appropriately.
If not maybe `api_batch_1` etc is better.

Reply:
There isn't any speciffic logic to the grouping. I changed it from `chunk` to `batch`.